### PR TITLE
J638 bump 394

### PR DESCRIPTION
**Pull request by grg121**
Closes https://github.com/wazuh/wazuh-jenkins/issues/638 by updating rpm SPECS to bump 3.9.4 version.

### CODE REVIEW

File: debs/SPECS/3.9.4/wazuh-agent/debian/changelog [hunk 1]
```diff
@@ -0,0 +1,241 @@
+wazuh-agent (3.9.3-RELEASE) stable; urgency=low
```
Comment (BraulioV) [inline debs/SPECS/3.9.4/wazuh-agent/debian/changelog]: The changelog lines form Wazuh v3.9.4 are missing.

File: debs/SPECS/3.9.4/wazuh-api/debian/changelog [hunk 2]
```diff
@@ -0,0 +1,159 @@
+wazuh-api (3.9.3-RELEASE) stable; urgency=low
```
Comment (BraulioV) [inline debs/SPECS/3.9.4/wazuh-api/debian/changelog]: The changelog lines form Wazuh v3.9.4 are missing.

File: debs/SPECS/3.9.4/wazuh-api/debian/control [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+Section: admin
+Priority: extra
+Maintainer: Wazuh, Inc <info@wazuh.com>
+Build-Depends: debhelper (>= 7.0.50~), nodejs (>= 4.6), npm, curl, gawk
+Standards-Version: 3.8.4
+Homepage: http://www.wazuh.com
+
+Package: wazuh-api
+Architecture: any
+Depends: ${shlibs:Depends}, npm, nodejs (>= 4.6.1), wazuh-manager (>= 3.9.3), wazuh-manager (<< 3.9.4), curl
```
Comment (BraulioV) [inline debs/SPECS/3.9.4/wazuh-api/debian/control]: This line must be updated to `wazuh-manager (>= 3.9.4), wazuh-manager (<< 3.9.5)`.

File: debs/SPECS/3.9.4/wazuh-manager/debian/changelog [hunk 4]
```diff
@@ -0,0 +1,172 @@
+wazuh-manager (3.9.3-RELEASE) stable; urgency=low
```
Comment (BraulioV) [inline debs/SPECS/3.9.4/wazuh-manager/debian/changelog]: The changelog lines form Wazuh v3.9.4 are missing.

File: rpms/SPECS/3.9.4/wazuh-api-3.9.4.spec [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+%attr(640, root, root) %{_localstatedir}/ossec/api/scripts/wazuh-api
+%attr(640, root, root) %{_localstatedir}/ossec/api/scripts/wazuh-api.service
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/api/node_modules
+%attr(750, ossec, ossec) %{_localstatedir}/ossec/api/node_modules/*
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/api/node_modules/.bin
+%{_localstatedir}/ossec/api/node_modules/.bin/*
+%dir %attr(750, ossec, ossec) %{_localstatedir}/ossec/logs/api
+%attr(660, ossec, ossec) %ghost %{_localstatedir}/ossec/logs/api.log
+
+%changelog
```
Comment (BraulioV) [inline rpms/SPECS/3.9.4/wazuh-api-3.9.4.spec]: The changelog lines form Wazuh v3.9.4 are missing.

File: rpms/SPECS/3.9.4/wazuh-manager-3.9.4.spec [hunk 6]
```diff
@@ -0,0 +1,793 @@
...
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/wodles/oscap
+%attr(750, root, ossec) %{_localstatedir}/ossec/wodles/oscap/oscap
+%attr(750, root, ossec) %{_localstatedir}/ossec/wodles/oscap/oscap.*
+%attr(750, root, ossec) %{_localstatedir}/ossec/wodles/oscap/template*
+%dir %attr(750, root, ossec) %{_localstatedir}/ossec/wodles/oscap/content
+%attr(640, root, ossec) %{_localstatedir}/ossec/wodles/oscap/content/*
+
+%{_initrddir}/*
+
+%changelog
```
Comment (BraulioV) [inline rpms/SPECS/3.9.4/wazuh-manager-3.9.4.spec]: The changelog lines form Wazuh v3.9.4 are missing.

File: solaris/solaris11/SPECS/template_agent_v3.9.4 copy.json [hunk 7]
```diff
@@ -0,0 +1,1635 @@
+{
```
Comment (BraulioV) [inline solaris/solaris11/SPECS/template_agent_v3.9.4 copy.json]: Rename this file to `template_agent_v3.9.4.json`

File: debs/SPECS/3.9.4/wazuh-agent/debian/changelog (hunk 1 above)
Comment (grg121) [reply debs/SPECS/3.9.4/wazuh-agent/debian/changelog]: https://github.com/wazuh/wazuh-packages/pull/228/commits/ca02cfa00eeb057c2e184ebf3ccea1922109ac1a

File: debs/SPECS/3.9.4/wazuh-api/debian/changelog (hunk 2 above)
Comment (grg121) [reply debs/SPECS/3.9.4/wazuh-api/debian/changelog]: https://github.com/wazuh/wazuh-packages/pull/228/commits/ca02cfa00eeb057c2e184ebf3ccea1922109ac1a

File: debs/SPECS/3.9.4/wazuh-api/debian/control (hunk 3 above)
Comment (grg121) [reply debs/SPECS/3.9.4/wazuh-api/debian/control]: https://github.com/wazuh/wazuh-packages/pull/228/commits/ca02cfa00eeb057c2e184ebf3ccea1922109ac1a

File: debs/SPECS/3.9.4/wazuh-manager/debian/changelog (hunk 4 above)
Comment (grg121) [reply debs/SPECS/3.9.4/wazuh-manager/debian/changelog]: https://github.com/wazuh/wazuh-packages/pull/228/commits/ca02cfa00eeb057c2e184ebf3ccea1922109ac1a

File: rpms/SPECS/3.9.4/wazuh-api-3.9.4.spec (hunk 5 above)
Comment (grg121) [reply rpms/SPECS/3.9.4/wazuh-api-3.9.4.spec]: https://github.com/wazuh/wazuh-packages/pull/228/commits/ca02cfa00eeb057c2e184ebf3ccea1922109ac1a

File: rpms/SPECS/3.9.4/wazuh-manager-3.9.4.spec (hunk 6 above)
Comment (grg121) [reply rpms/SPECS/3.9.4/wazuh-manager-3.9.4.spec]: https://github.com/wazuh/wazuh-packages/pull/228/commits/ca02cfa00eeb057c2e184ebf3ccea1922109ac1a

File: solaris/solaris11/SPECS/template_agent_v3.9.4 copy.json (hunk 7 above)
Comment (grg121) [reply solaris/solaris11/SPECS/template_agent_v3.9.4 copy.json]: https://github.com/wazuh/wazuh-packages/pull/228/commits/2c323a91ed29fa10c8e1732e40b7f3665067c08e

**Comment by grg121** [comment]
Required changes done.

**Pull request merged**